requestor-tutorials/vm-runtime/computation-payload-manifest.md

```diff
@@ -0,0 +1,278 @@
+---
+description: Computation Payload Manifest description, its schema, and configuration guide
+---
+
+# Computation Payload Manifest
+
+_Computation Payload Manifest_ allows [Requestor](../../introduction/requestor.md) to define application package _Payload_ and allows to set constraints on _Computations_ performed on [Provider](../../introduction/provider.md) node.
+
+Request with a manifest can be [configured in yapapi](#configuration). 
```

Comment:
"Request"? maybe "Payload" ?

Reply:
and then, of course, "A payload with..."

Reply:
If anything I would choose word "Demand" instead of a "Payload". Compuation PAYLOAD Manifest describes a PAYLOAD which is a fraction of something bigger. I just named it a "Request".
We should have some strict dictionary how to name these things.